### PR TITLE
update with new content packs

### PR DESCRIPTION
**Pull request by jnhunsberger**
### What does this PR do? What is the motivation?

Need to update the content pack documentation with new content packs.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/security/cloud_siem/content_packs.md [hunk 1]
```diff
@@ -25,9 +25,10 @@ further_reading:
...
-    {{< nextlink href="/security/cloud_siem/content_packs#authentication-content-packs" >}}<u>Authentication</u>: 1Password, Auth0, JumpCloud, Okta{{< /nextlink >}}
-    {{< nextlink href="/security/cloud_siem/content_packs#collaboration-content-packs" >}}<u>Collaboration</u>: Google Workspace, Microsoft 365{{< /nextlink >}}
-    {{< nextlink href="/security/cloud_siem/content_packs#network-content-packs" >}}<u>Network</u>: Cloudflare{{< /nextlink >}}
+    {{< nextlink href="/security/cloud_siem/content_packs#authentication-content-packs" >}}<u>Authentication</u>: 1Password, Auth0, Cisco DUO, JumpCloud, Okta{{< /nextlink >}}
+    {{< nextlink href="/security/cloud_siem/content_packs#collaboration-content-packs" >}}<u>Collaboration</u>: Google Workspace, Microsoft 365, Slack Audit Logs{{< /nextlink >}}
+    {{< nextlink href="/security/cloud_siem/content_packs#network-content-packs" >}}<u>Network</u>: Cloudflare, Cisco Meraki, Cisco Umbrella, Palo Alto Networks Firewall{{< /nextlink >}}
+    {{< nextlink href="/security/cloud_siem/content_packs#web-security-content-packs" >}}<u>Network</u>: nginx{{< /nextlink >}}
```
Comment (estherk15) [inline content/en/security/cloud_siem/content_packs.md:31]:
```suggestion
    {{< nextlink href="/security/cloud_siem/content_packs#web-security-content-packs" >}}<u>Network</u>: NGINX{{< /nextlink >}}
```

File: content/en/security/cloud_siem/content_packs.md [hunk 2]
```diff
@@ -135,6 +152,40 @@ The [Cloudflare Content Pack][25] includes:
...
+
+Collect and monitor logs from Cisco Umbrella to gain insights into DNS and Proxy logs
+
+The [Cisco Umbrella Content Pack][39] includes:
+- [Detection Rules][40]
+- An interactive dashboard
+
+## Web Security Content Packs
+
+### nginx
```
Comment (estherk15) [inline content/en/security/cloud_siem/content_packs.md:181]:
```suggestion
### NGINX
```

File: content/en/security/cloud_siem/content_packs.md [hunk 3]
```diff
@@ -135,6 +152,40 @@ The [Cloudflare Content Pack][25] includes:
...
+
+Monitor and respond to web-based risks with Nginx
```
Comment (estherk15) [inline content/en/security/cloud_siem/content_packs.md:183]:
```suggestion
Monitor and respond to web-based risks with NGINX
```

File: content/en/security/cloud_siem/content_packs.md [hunk 4]
```diff
@@ -135,6 +152,40 @@ The [Cloudflare Content Pack][25] includes:
...
+
+The [nginx Content Pack][41] includes:
```
Comment (estherk15) [inline content/en/security/cloud_siem/content_packs.md:185]:
```suggestion
The [NGINX Content Pack][41] includes:
```

**Review by estherk15 — APPROVED** [review]
Thanks @jnhunsberger for adding content pack information! Suggest changing NGINX to consistent capitalization. Let me know when this is good to merge!

File: content/en/security/cloud_siem/content_packs.md (hunk 4 above)
Comment (jnhunsberger) [reply content/en/security/cloud_siem/content_packs.md:185]: All good. Fully support the changes.

**Comment by jnhunsberger** [comment]
The changes look good to me.

**Pull request merged**